packages/js/manifests/polywrap/scripts/templates/migrate-ts.mustache

```diff
@@ -31,13 +31,18 @@ export function migrate{{#latest}}{{type}}{{/latest}}(
   manifest: Any{{#latest}}{{type}}{{/latest}},
   to: {{#latest}}{{type}}{{/latest}}Formats
 ): {{#latest}}{{type}}{{/latest}} {
-  const from = manifest.format as {{#latest}}{{type}}{{/latest}}Formats;
+  let from = manifest.format as {{#latest}}{{type}}{{/latest}}Formats;
+
+  // HACK: Patch fix for backwards compatability
```

Comment:
Is this hack temporary or do u think it will stay for the long term? If is the latter if would remove the hack word and just add the comment to make it cleaner

Reply:
Yah I think we can potentially get rid of this in the near future, since the `0.1` format was only "live" for a very short time so if there are no published wrappers with this version we can remove this backwards compatibility check in the future.

Also we'll remove all back-compat when we reach `1.0.0`.